lib/core/http.js

```diff
@@ -61,5 +61,9 @@ module.exports = (config) => {
     });
   }
 
-  return { getCageKey, runCage, getCert };
+  const reportMetric = () => {
+    return post(config.metricsUrl, '', { event: 'Data Encrypted' })
```

Comment:
```suggestion
    return post(config.metricsUrl, { event: 'Data Encrypted' })
```

Reply:
Is this the wrong way around and should be above?

Reply:
Yes, it's the opposite in the client side SDK, will commit the change